ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
I will install CMake 2.8.3 and try it. All our build systems have CMake 2.8.11+.

For dependencie there are the ones from scm, for example ws_workunits.ecm, ws_packageprocess.ecm. These are fine since scm built with Clienttools and espscm is added to the targets' dependencies. The problem comes from esp/service/<ws_workunits|ws_packageprocess> which are only built with "Platform" currently. Clienttools eclcmd, etc depend on esp/service/<ws_workunits|ws_packageprocess>.